docs/en/pid-eaa-data-model.rst

```diff
@@ -16,11 +16,9 @@ The User attributes carried within the Italian PID are the ones listed below:
     - Unique Identifier
     - Taxpayer identification number
 
-The italian PID is extended according to the `OpenID Identity Assurance Profile [OIDC.IDA] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html>`_, that enables the binding of the PID to a national trust framework, giving all the evidence of the identity proofing procedures underlying the PID issuing in both remote and proximity flows.
+The italian PID contains additional claims and according to the `OpenID Identity Assurance Profile [OIDC.IDA] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html>`_, these carries the national trust framework and the identity proofing procedures underlying the issuance of the PID. In particular, the PID carries some relevant information about the Authentic Sources of the subject's attributes. 
 
-The (Q)EAAs are issued by (Q)EAA Issuers to a Wallet Instance and MUST be provided in SD-JWT-VC or mDOC CBOR data format. 
-
-The (Q)EAAs are extended according to the `OpenID Identity Assurance Profile [OIDC.IDA] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html>`_, that allows the recipients to know the Authentic Sources where the data comes from. 
+The (Q)EAAs are issued by (Q)EAA Issuers to a Wallet Instance and MUST be provided in SD-JWT-VC or mDOC CBOR data format.  
```

Comment:
Why do you remove the IDA reference here? A (Q)EAA data model is also extended with IDA, am I missing something?

Reply:
if both PID and (Q)EAA uses IDA, it would be enough write this a single time in the introduction text